parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -109,6 +109,13 @@ ERR_CREDENTIAL_PROVIDER_DOWNLOAD_TIMEOUT=205 # Timeout waiting for credential pr
 
 ERR_CNI_VERSION_INVALID=206 # reference CNI (not azure cni) needs a valid version in components.json
 
+ERR_ORAS_PULL_K8S_FAIL=207 # Error pulling kube-node artifact via oras from registry
+ERR_ORAS_PULL_FAIL_RESERVE_1=208 # Error pulling artifact with oras from registry
```

Comment:
what are these `_*` exit codes for?

Reply:
they are reserved for other binaries like azure-acr-credentials-provider, cni, containerd-shim-wsam.
Reserved now to make these exit code number to be consecutive